src/Engine/ProtoCore/DSASM/DSasmDefs.cs

```diff
@@ -450,7 +450,7 @@ public struct Constants
         public const string kTempProcConstant = "temp_proc_var_";
         public const string kTempProcLeftVar = "%" + kTempProcConstant;
         public const string kImportData = "ImportData";
-        public const string kTempVarForNonAssignment = "temp_";
+        public const string kTempVarForNonAssignment = "temp6BBA4B28C5E54CF89F300D510499A00E_";
```

Comment:
Because `kTempVarForNonAssignment` is used for expression without left-hand side variable. For example, `1;` will be converted to `temp_1 = 1;`. But it has problem: if there is a `temp_1` in the code, it will conflict with that one. See the picture below, note no output port for `temp_1`.
![untitled](https://cloud.githubusercontent.com/assets/2600379/7270273/969f0efc-e90d-11e4-97ed-194a66c21ab2.png)